### ISSUE TITLE
The top side of the Slime Block looks incorrect

### ISSUE THREAD

**Issue by JustJabka** [misode/misode.github.io#593]
![image](https://github.com/user-attachments/assets/571b16c3-17c1-4e56-b70f-2aa477d59142)

**Comment by misode** [comment]
Hmm I cannot reproduce this... Maybe it was a weird caching issue? @JustJabka Are you still experiencing this problem?
![image](https://github.com/user-attachments/assets/64ba33c5-6959-4c98-bd63-9fa81270ec7d)

**Comment by JustJabka** [comment]
Yes, I'm still experiencing this problem, I tried clearing my browser cache, switching to a light theme, but nothing changed.

**Comment by misode** [comment]
@JustJabka Have you tried using another browser or other device? Does this happen everywhere? What browser are you using?

**Comment by JustJabka** [comment]
@misode I checked it on my phone and there was no such problem. I noticed that this problem occurs not only with the slime block, but with all semi-transparent blocks. I use the firefox browser.

**Comment by tryashtar** [comment]
I have this same issue on both PC and mobile, both using firefox

**Comment by neylz** [comment]
also affects honey blocks

<img width="425" height="61" alt="Image" src="https://github.com/user-attachments/assets/c8645f7f-a8bf-4b3c-a0d2-7b472a97d19d" />
